### PR TITLE
Fixed: UI of returnDetails page(#2v7gpv1)

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/views/PurchaseOrderDetail.vue [hunk 1]
```diff
@@ -270,18 +270,18 @@ export default defineComponent({
 </script>
 
 <style scoped>
-.po-meta {
+/* .po-meta {
```
Comment (dt2patel) [inline src/views/PurchaseOrderDetail.vue:273]: need to delete commented styles

File: src/views/ReturnDetails.vue [hunk 2]
```diff
@@ -46,37 +47,42 @@
...
               {{ $t("Receive All") }}
             </ion-button>
             <ion-progress-bar :value="item.quantityAccepted/item.quantityOrdered" />
             <p slot="end">{{ item.quantityOrdered }} {{ $t("returned") }}</p>
           </ion-item>
-
         </ion-card>
       </main>
 
+      <!-- <div> -->
```
Comment (dt2patel) [inline src/views/ReturnDetails.vue:79]: delete commented div

**Review by dt2patel — CHANGES_REQUESTED** [review]
remove commented code

File: src/views/PurchaseOrderDetail.vue (hunk 1 above)
Comment (shashwatbangar) [reply src/views/PurchaseOrderDetail.vue:273]: Done sir

**Review by dt2patel — APPROVED** [review]

**Pull request merged**